test/integration/projects/projects_test.go

```diff
@@ -159,9 +159,8 @@ func TestProjects(t *testing.T) {
 							assert.Subset(listApis, restrictedApisEnabled, "APIs should have been enabled")
 
 							restrictedProjectNumber := projects.GetStringOutput("restricted_shared_vpc_project_number")
-							perimeter := gcloud.Runf(t, "access-context-manager perimeters describe %s --policy %s", perimeterName, policyID)
-							listResources := utils.GetResultStrSlice(perimeter.Get("status.resources").Array())
-							assert.Contains(listResources, fmt.Sprintf("projects/%s", restrictedProjectNumber), "restricted project should be in the perimeter")
+							perimeter, err := gcloud.RunCmdE(t, fmt.Sprintf("access-context-manager perimeters dry-run describe %s --policy %s", perimeterName, policyID))
+							assert.True(strings.Contains(perimeter, restrictedProjectNumber), fmt.Sprintf("dry-run service perimeter %s should contain project %s", perimeterName, restrictedProjectNumber)
```

Comment:
```suggestion
							perimeter, err := gcloud.RunCmdE(t, fmt.Sprintf("access-context-manager perimeters dry-run describe %s --policy %s", perimeterName, policyID))
							assert.NoError(err)
							assert.True(strings.Contains(perimeter, restrictedProjectNumber), fmt.Sprintf("dry-run service perimeter %s should contain project %s", perimeterName, restrictedProjectNumber))
```

@eeaton this will fix the compilation issues:

A) Missing closing `)`
```
Step #2 - "create-bootstrap": Error: /workspace/test/integration/projects/projects_test.go:163:183: missing ',' before newline in argument list (and 106 more errors)
```

and `err` var not used

```
./projects_test.go:162:19: err declared and not used
```